test/rules/assertions/checkLineAlignment.js

```diff
@@ -1057,6 +1057,36 @@ export default {
       const fn = ( lorem, sit ) => {}
       `,
     },
+    {
+      code: `\r
+        /**\r
+         * Function description.\r
+         *\r
+         * @param {string} lorem Description.\r
+         * @param {int} sit Description multi words.\r
```

Comment:
I'd suggest at least test manually (if you don't want to expand this test), making one of the descriptions multiline, and add an empty line and then a `@return` tag. Just to make sure we didn't miss any situation. :)